privacyidea/lib/eventhandler/usernotification.py

```diff
@@ -224,6 +248,7 @@ def do(self, action, options=None):
         handler_def = options.get("handler_def")
         handler_options = handler_def.get("options", {})
         notify_type = handler_options.get("To", NOTIFY_TYPE.TOKENOWNER)
+        reply_to_type = handler_options.get("reply_to", NOTIFY_TYPE.TOKENOWNER)
```

Comment:
Old installations do not have the "type" in ` reply_to`  but the destinct email address!